unit-test/fm_child_tests.c

```diff
@@ -3512,6 +3512,64 @@ void Test_FM_ChildLoop_CountSemTakeSuccessDefault(void)
  * ChildSleepStat Tests
  * ***************/
 
+void Test_FM_ChildSleepStat_getSizeTimeModeFalse(void)
+{
+    // Arrange
+    const char *dummy_filename          = "dummy_filename";
+    FM_DirListEntry_t dummy_DirListData = {.EntrySize=1, .ModifyTime=1, .Mode=1};
```

Comment:
To avoid future uninitialized variable warnings if elements are added to the structure I'd recommend just memsetting to 0 and then set the elements you care about.

Reply:
Comment only applies to line 3519.